arionBackend/transformation/esper/esper_transformer.py

```diff
@@ -0,0 +1,15 @@
+from arionBackend.transformation.transformator import Transformer
+
+
+class EsperTransformer(Transformer):
+    """
+    This class extends the abstract base class Transformer and implements a transformer for Esper EPL.
+    """
+
+    def transform(self, queries):
+        """
+        This method transform an Esper EPL query into a hierarchy.
+        :param queries: Esper EPL queries
+        :return: 

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1131)